src/openforms/submissions/tests/test_admin_email_verifications.py

```diff
@@ -0,0 +1,25 @@
+from django.urls import reverse
+
+from django_webtest import WebTest
+from maykin_2fa.test import disable_admin_mfa
+
+from openforms.accounts.tests.factories import SuperUserFactory
+
+from .factories import EmailVerificationFactory
+
+
+@disable_admin_mfa()
+class EmailVerificationAdminTests(WebTest):
+
+    def test_search_fields(self):
+        # smoke test that the search fields are correctly configured
+        user = SuperUserFactory.create()
+        url = reverse("admin:submissions_emailverification_changelist")
+        EmailVerificationFactory.create_batch(3)
+        EmailVerificationFactory.create(verified=True, email="openforms@opengem.nl")
```

Comment:
Maybe we should use another email for testing purposes?

Reply:
what do you mean? I'm using (part of) this email to apply the search in the admin

Reply:
discussed on slack :)